### PR TITLE
Add chromium-wayland-vaapi patches

### PR DESCRIPTION
**Pull request by EliasTheGrandMasterOfMistakes**

### CODE REVIEW

File: patches/chromium/0001-enable-linux-unstable-deb-target.patch
Comment (bbhtt) [inline patches/chromium/0001-enable-linux-unstable-deb-target.patch:1]: this patch file looks broken

Comment (EliasTheGrandMasterOfMistakes) [reply patches/chromium/0001-enable-linux-unstable-deb-target.patch:1]: oh sorry is my fault, this force to build with embedded minigbm
i have downloaded with error, i will replace.
thanks for advise

**Comment by EliasTheGrandMasterOfMistakes** [comment]
Sorry for the commits name if this build works, i will rebase and rename commits and remove the DO NOT MERGE commit

**Comment by EliasTheGrandMasterOfMistakes** [comment]
i think this commit makes nothing difference, can be hard to build using systemside ffmpeg libraries, in this case will be necessary to add ffmpeg-full extensions and fedora patches for build using system side, one day i will try this

**Comment by EliasTheGrandMasterOfMistakes** [comment]
image
this needs to work, why good WHY!!!!

**Comment by EliasTheGrandMasterOfMistakes** [comment]
is necessary to rebase this patch, i will try after buy a new hd because chromium repo is much bigger

**Comment by EliasTheGrandMasterOfMistakes** [comment]
i'm just in a minimalist ssd

**Comment by EliasTheGrandMasterOfMistakes** [comment]
Is building but in my igpu not working, idk if in new igpu is working, is necessary some testing with ozone platform wayland
-enable-features=VaapiVideoDecodeLinuxGL is the new flag for vaapi in opengl backend

**Comment by ninja-** [comment]
bot, build

**Comment by ninja-** [comment]
I'll be happy to test this for you with AMD and unpatched mesa :) in the meantime, can we get maintainers perspective on merging this?

**Comment by bbhtt** [comment]
Why are you queuing builds? Chromium takes over 3 hours to build on Flathub

**Comment by ninja-** [comment]
The build expired:

➜  ~ flatpak install --user https://dl.flathub.org/build-repo/80786/org.chromium.Chromium.flatpakref
błąd: Can't load uri https://dl.flathub.org/build-repo/80786/org.chromium.Chromium.flatpakref: Server returned status 404

I believe these test builds have limited lifetime

**Comment by Erick555** [comment]
> in the meantime, can we get maintainers perspective on merging this?

I think maintainer shared their perspective on this [before](https://github.com/flathub/org.chromium.Chromium/pull/299#issuecomment-1598105884). This is n-th time similar PR is submitted here.

**Comment by EliasTheGrandMasterOfMistakes** [comment]
> I'll be happy to test this for you with AMD and unpatched mesa :) in the meantime, can we get maintainers perspective on merging this? New Chromium will be released soon, but I believe you only then need to drop one commit that was pulled 1:1 into Chromium lately. The rest of patches are still relevant for Wayland to work and work with unpatched mesa

Sadly this looks not works well in RadeonSI VAAPI driver and doens't works i965 VAAPI driver, sadly VAAPI for chromium works on top multiplanar formats and have amount of problems.
The official maintainer looks not have plans to merge it, and Chromium is already merging some necessary patches, i think in the next release VAAPI will work in AMD and intel olders gpus

**Comment by EliasTheGrandMasterOfMistakes** [comment]
I'm closing, this patches just working in newest intel GPUs and can't work in AMD, the lastest patch for enable ozone wayland in build is already in upstream, is just a little time question to all others patches will be merged too.

**Comment by ninja-** [comment]
> is just a little time question to all others patches will be merged too.

I doubt that, Google has no intention of fixing VA-API on Wayland and it will probably stay broken for nearest future.
Maintainers of different package system will have more or less intention to maintain the right patches going forward.

> Sadly this looks not works well in RadeonSI VAAPI driver and doens't works i965 VAAPI driver, sadly VAAPI for chromium works on top multiplanar formats and have amount of problems.

I thought that giga patch for disjoint VAAPI fixed that?

**Pull request closed without merging**